dotnet/src/webdriver/SeleniumManager.cs

```diff
@@ -214,6 +186,27 @@ public static string DriverPath(DriverOptions options)
                 throw new WebDriverException($"Error deserializing Selenium Manager's response: {output}", ex);
             }
 
+            if (result.ContainsKey("logs"))
+            {
+                Dictionary<string, string> logs = result["logs"] as Dictionary<string, string>;
+                foreach (KeyValuePair<string, string> entry in logs)
+                {
+                    switch (entry.Key)
+                    {
+                        case "WARN":
+                            _logger.Warn(entry.Value);
```

Comment:
```csharp
if (_logger.IsEnabled(Warning)) { // log warning }
```

Currently this pattern should be followed to keep internal logging performant and less memory allocation.

Reply:
OK, I will change it.